packages/recommend-core/src/getTrendingFacets.ts

```diff
@@ -17,6 +18,12 @@ export type TrendingFacetsProps<TObject> = {
   transformItems?: (
     items: Array<TrendingFacet<TObject>>
   ) => Array<TrendingFacet<TObject>>;
+  /**
+   * A prop that allows the user to provide initial results from outside
+   *
+   * It's useful for building SSR on top of Recommend
+   */
+  initialState?: InitialResults<TObject>;
```

Comment:
This prop shouldn't be part of the static get methods, but only in our Hooks.

Reply:
✅